runtime/src/in_mem_accounts_index.rs

```diff
@@ -1298,4 +1349,140 @@ mod tests {
         assert!(test.get_should_age(test.storage.current_age()));
         assert_eq!(test.storage.count_ages_flushed(), 0);
     }
+
+    #[test]
+    fn test_update_slot_list_other() {
+        solana_logger::setup();
+        let mut reclaims = Vec::default();
+        let previous_slot_entry_was_cached = false;
+        let new_slot = 0;
+        let info = 1;
+        let other_value = info + 1;
+        let at_new_slot = (new_slot, info);
+        let unique_other_slot = new_slot + 1;
+        for other_slot in [Some(new_slot), Some(unique_other_slot), None] {
+            let mut slot_list = Vec::default();
+            // upserting into empty slot_list, so always addref
+            assert!(
+                InMemAccountsIndex::update_slot_list(
+                    &mut slot_list,
+                    new_slot,
+                    info,
+                    other_slot,
+                    &mut reclaims,
+                    previous_slot_entry_was_cached
+                ),
+                "other_slot: {:?}, ",
```

Comment:
nit: I'm not sure what this text here is meant to communicate. Is it leftover from debugging/implementing?

Reply:
fixed